hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientInvocationFuture.java

```diff
@@ -72,6 +72,8 @@ public boolean isDone() {
 
     @Override
     public ClientMessage get() throws InterruptedException, ExecutionException {
+        //Client should wait remote invocation indefinitely on internal thread
```

Comment:
This comment doesn't make sense. The comments states 'it shuold wait indefinitely on internal thread', but the assert makes sure it isn't an internal thread.